requirements-provider.txt

```diff
@@ -8,7 +8,7 @@ GDAL<=3.8.4
 geoalchemy2
 netCDF4
 numpy
-oracledb
+oracledb==2.2.0
```

Comment:
@8luewater Do we need to pin to 2.2.0 or can we default to latest (2.2.1)?  I would prefer having unpinned dependencies in master for providers to keep us current.